wsl-pro-service/internal/system/system_test.go

```diff
@@ -22,6 +22,41 @@ const (
 	mockBadOutput                      // A mock that returns a bad value with no error
 )
 
+func TestNew(t *testing.T) {
+	t.Parallel()
+
+	testCases := map[string]struct {
+		systemLandscapeConfigFile string
+	}{
+		"Return a new system": {},
+
+		"Only prints a warning when the landscape config validation failed": {systemLandscapeConfigFile: "invalid_ini.conf"},
```

Comment:
I don't see where we assert the warning is printed, as the test case name suggests. Can you clarify?

Reply:
We can’t capture logs with the current test setup apart of not running this subtest in parallel. What I wanted to emphasize was that we don’t return an error (and we can see logs when running the tests) and describe its behaviour. I can capture it and ensure we print something but that seems overkill. WDYT?

Reply:
Yeap, overkill certainly. 

I'd just rephrase the title to enforce that what we are checking here is the absence of errors: "Sucess when Landscape config validation failed (only warnings)". I know "Success" here is less precise, "No errors when ..." is not the best alternative because it starts with a negation, "(only warnings)" in parenthesis (or commas) should de-emphasize it compared to have that in the beginning of the sentence. If alternatives like that don't seem to capture the message you want to pass, then we should add a comment in the end of test body so we don't think about the same question in the future  :)

Reply:
What about `Ignore errors when the landscape config validation failed (only warnings)`?